2020-05-09 16:27:39.137 [info] <0.43.0> Application lager started on node nonode@nohost
2020-05-09 16:27:39.192 [info] <0.43.0> Application xmerl started on node nonode@nohost
2020-05-09 16:27:39.292 [info] <0.43.0> Application sasl started on node nonode@nohost
2020-05-09 16:27:39.336 [info] <0.43.0> Application tools started on node nonode@nohost
2020-05-09 16:27:39.351 [info] <0.43.0> Application jsx started on node nonode@nohost
2020-05-09 16:27:39.368 [info] <0.43.0> Application recon started on node nonode@nohost
2020-05-09 16:27:39.815 [info] <0.43.0> Application credentials_obfuscation started on node nonode@nohost
2020-05-09 16:27:39.815 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2020-05-09 16:27:39.830 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2020-05-09 16:27:39.831 [info] <0.43.0> Application amqp started on node nonode@nohost
2020-05-09 16:27:39.843 [info] <0.43.0> Application connection started on node nonode@nohost
2020-05-09 16:27:39.850 [info] <0.43.0> Application decimal started on node nonode@nohost
2020-05-09 16:27:39.878 [info] <0.43.0> Application db_connection started on node nonode@nohost
2020-05-09 16:27:39.902 [info] <0.43.0> Application postgrex started on node nonode@nohost
2020-05-09 16:27:39.926 [info] <0.43.0> Application eex started on node nonode@nohost
2020-05-09 16:27:40.053 [info] <0.43.0> Application telemetry started on node nonode@nohost
2020-05-09 16:27:40.080 [info] <0.43.0> Application ecto started on node nonode@nohost
2020-05-09 16:27:40.086 [info] <0.43.0> Application ecto_sql started on node nonode@nohost
2020-05-09 16:27:40.138 [info] <0.43.0> Application mime started on node nonode@nohost
2020-05-09 16:27:40.149 [info] <0.43.0> Application plug_crypto started on node nonode@nohost
2020-05-09 16:27:40.162 [info] <0.43.0> Application plug started on node nonode@nohost
2020-05-09 16:27:40.172 [info] <0.43.0> Application plug_cowboy started on node nonode@nohost
2020-05-09 16:27:40.198 [error] <0.753.0> CRASH REPORT Process <0.753.0> with 0 neighbours exited with reason: {#{'__exception__' => true,'__struct__' => 'Elixir.UndefinedFunctionError',arity => 1,function => init,message => nil,module => 'Elixir.User.Plug.Router',reason => nil},[{'Elixir.User.Plug.Router',init,[[]],[]},{'Elixir.Plug.Cowboy',dispatch_for,2,[{file,"lib/plug/cowboy.ex"},{line,305}]},{'Elixir.Plug.Cowboy',to_args,5,[{file,"lib/plug/cowboy.ex"},{line,244}]},{'Elixir.Plug.Cowboy',child_spec,1,[{file,"lib/plug/cowboy.ex"},{line,170}]},{'Elixir.Supervisor',init_child,1,[{file,"lib/supervis..."},...]},...]} in application_master:init/4 line 138
2020-05-09 16:28:07.951 [info] <0.43.0> Application lager started on node nonode@nohost
2020-05-09 16:28:07.977 [info] <0.43.0> Application xmerl started on node nonode@nohost
2020-05-09 16:28:08.032 [info] <0.43.0> Application sasl started on node nonode@nohost
2020-05-09 16:28:08.062 [info] <0.43.0> Application tools started on node nonode@nohost
2020-05-09 16:28:08.072 [info] <0.43.0> Application jsx started on node nonode@nohost
2020-05-09 16:28:08.084 [info] <0.43.0> Application recon started on node nonode@nohost
2020-05-09 16:28:08.677 [info] <0.43.0> Application credentials_obfuscation started on node nonode@nohost
2020-05-09 16:28:08.677 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2020-05-09 16:28:08.714 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2020-05-09 16:28:08.726 [info] <0.43.0> Application amqp started on node nonode@nohost
2020-05-09 16:28:08.771 [info] <0.43.0> Application connection started on node nonode@nohost
2020-05-09 16:28:08.798 [info] <0.43.0> Application decimal started on node nonode@nohost
2020-05-09 16:28:08.833 [info] <0.43.0> Application db_connection started on node nonode@nohost
2020-05-09 16:28:08.867 [info] <0.43.0> Application postgrex started on node nonode@nohost
2020-05-09 16:28:08.908 [info] <0.43.0> Application eex started on node nonode@nohost
2020-05-09 16:28:08.934 [info] <0.43.0> Application telemetry started on node nonode@nohost
2020-05-09 16:32:55.275 [info] <0.43.0> Application lager started on node nonode@nohost
2020-05-09 16:32:55.302 [info] <0.43.0> Application xmerl started on node nonode@nohost
2020-05-09 16:32:55.359 [info] <0.43.0> Application sasl started on node nonode@nohost
2020-05-09 16:32:55.382 [info] <0.43.0> Application tools started on node nonode@nohost
2020-05-09 16:32:55.389 [info] <0.43.0> Application jsx started on node nonode@nohost
2020-05-09 16:32:55.399 [info] <0.43.0> Application recon started on node nonode@nohost
2020-05-09 16:32:55.766 [info] <0.43.0> Application credentials_obfuscation started on node nonode@nohost
2020-05-09 16:32:55.766 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2020-05-09 16:32:55.779 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2020-05-09 16:32:55.783 [info] <0.43.0> Application amqp started on node nonode@nohost
2020-05-09 16:32:55.795 [info] <0.43.0> Application connection started on node nonode@nohost
2020-05-09 16:32:55.803 [info] <0.43.0> Application decimal started on node nonode@nohost
2020-05-09 16:32:55.823 [info] <0.43.0> Application db_connection started on node nonode@nohost
2020-05-09 16:32:55.843 [info] <0.43.0> Application postgrex started on node nonode@nohost
2020-05-09 16:32:55.861 [info] <0.43.0> Application eex started on node nonode@nohost
2020-05-09 16:32:55.876 [info] <0.43.0> Application telemetry started on node nonode@nohost
2020-05-09 16:32:55.888 [info] <0.43.0> Application ecto started on node nonode@nohost
2020-05-09 16:32:55.890 [info] <0.43.0> Application ecto_sql started on node nonode@nohost
2020-05-09 16:32:55.916 [info] <0.43.0> Application mime started on node nonode@nohost
2020-05-09 16:32:55.921 [info] <0.43.0> Application plug_crypto started on node nonode@nohost
2020-05-09 16:32:55.927 [info] <0.43.0> Application plug started on node nonode@nohost
2020-05-09 16:32:55.932 [info] <0.43.0> Application plug_cowboy started on node nonode@nohost
2020-05-09 16:32:56.002 [info] <0.43.0> Application conference started on node nonode@nohost
2020-05-09 20:17:19.620 [error] <0.383.0> gen_server <0.383.0> terminated with reason: {socket_error,timeout}
2020-05-09 20:17:19.658 [error] <0.378.0> gen_server <0.378.0> terminated with reason: {socket_error,timeout}
2020-05-09 20:17:21.521 [error] <0.378.0> CRASH REPORT Process <0.378.0> with 0 neighbours exited with reason: {socket_error,timeout} in gen_server:handle_common_reply/8 line 751
2020-05-09 20:17:21.555 [error] <0.376.0> Supervisor {<0.376.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.377.0>, {amqp_params_network,<<"guest">>,<<"jwCOBLkSqFXmiA+/vmtiUa9vznwLqAgnQSnsmwvu0+F5W4PjzlQHi/WgVNwAnq...">>,...}) at <0.378.0> exit with reason {socket_error,timeout} in context child_terminated
2020-05-09 20:17:21.556 [error] <0.376.0> Supervisor {<0.376.0>,amqp_connection_sup} had child connection started with amqp_gen_connection:start_link(<0.377.0>, {amqp_params_network,<<"guest">>,<<"jwCOBLkSqFXmiA+/vmtiUa9vznwLqAgnQSnsmwvu0+F5W4PjzlQHi/WgVNwAnq...">>,...}) at <0.378.0> exit with reason reached_max_restart_intensity in context shutdown
2020-06-06 21:58:29.056 [info] <0.44.0> Application lager started on node nonode@nohost
2020-06-06 21:58:29.108 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-06-06 21:58:29.193 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-06-06 21:58:29.222 [info] <0.44.0> Application tools started on node nonode@nohost
2020-06-06 21:58:29.228 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-06-06 21:58:29.280 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-06-06 21:58:29.287 [info] <0.44.0> Application recon started on node nonode@nohost
2020-06-06 21:58:29.478 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-06-06 21:58:29.478 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-06-06 21:58:29.550 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-06-06 21:58:29.553 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-06-06 21:58:29.563 [info] <0.44.0> Application poison started on node nonode@nohost
2020-06-06 21:58:35.797 [info] <0.44.0> Application prometheus started on node nonode@nohost
2020-06-06 21:58:35.797 [info] <0.44.0> Application prometheus_ex started on node nonode@nohost
2020-06-06 21:58:35.805 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-06-06 21:58:35.850 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-06-06 21:58:35.854 [info] <0.44.0> Application accept started on node nonode@nohost
2020-06-06 21:58:35.865 [info] <0.44.0> Application connection started on node nonode@nohost
2020-06-06 21:58:35.872 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-06-06 21:58:35.882 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-06-06 21:58:35.887 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-06-06 21:58:35.907 [info] <0.44.0> Application eex started on node nonode@nohost
2020-06-06 21:58:35.961 [info] <0.44.0> Application telemetry started on node nonode@nohost
2020-06-06 21:58:35.971 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-06-06 21:58:35.974 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-06-06 21:58:36.157 [info] <0.44.0> Application jose started on node nonode@nohost
2020-06-06 21:58:36.157 [info] <0.44.0> Application joken started on node nonode@nohost
2020-06-06 21:58:36.171 [info] <0.44.0> Application mime started on node nonode@nohost
2020-06-06 21:58:36.177 [info] <0.44.0> Application plug_crypto started on node nonode@nohost
2020-06-06 21:58:36.181 [info] <0.44.0> Application plug started on node nonode@nohost
2020-06-06 21:58:36.184 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-06-06 21:58:36.186 [info] <0.44.0> Application jsonapi started on node nonode@nohost
2020-06-06 21:58:36.186 [info] <0.44.0> Application prometheus_plugs started on node nonode@nohost
2020-06-06 21:58:36.980 [info] <0.44.0> Application conference started on node nonode@nohost
2020-06-06 21:59:21.924 [info] <0.44.0> Application lager started on node nonode@nohost
2020-06-06 21:59:21.939 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-06-06 21:59:21.966 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-06-06 21:59:21.981 [info] <0.44.0> Application tools started on node nonode@nohost
2020-06-06 21:59:21.987 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-06-06 21:59:21.999 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-06-06 21:59:22.006 [info] <0.44.0> Application recon started on node nonode@nohost
2020-06-06 21:59:22.142 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-06-06 21:59:22.142 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-06-06 21:59:22.150 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-06-06 21:59:22.153 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-06-06 21:59:22.160 [info] <0.44.0> Application poison started on node nonode@nohost
2020-06-06 21:59:24.045 [info] <0.44.0> Application prometheus started on node nonode@nohost
2020-06-06 21:59:24.045 [info] <0.44.0> Application prometheus_ex started on node nonode@nohost
2020-06-06 21:59:24.053 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-06-06 21:59:24.059 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-06-06 21:59:24.063 [info] <0.44.0> Application accept started on node nonode@nohost
2020-06-06 21:59:24.075 [info] <0.44.0> Application connection started on node nonode@nohost
2020-06-06 21:59:24.082 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-06-06 21:59:24.092 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-06-06 21:59:24.097 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-06-06 21:59:24.110 [info] <0.44.0> Application eex started on node nonode@nohost
2020-06-06 21:59:24.122 [info] <0.44.0> Application telemetry started on node nonode@nohost
2020-06-06 21:59:24.132 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-06-06 21:59:24.134 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-06-06 21:59:24.210 [info] <0.44.0> Application jose started on node nonode@nohost
2020-06-06 21:59:24.210 [info] <0.44.0> Application joken started on node nonode@nohost
2020-06-06 21:59:24.224 [info] <0.44.0> Application mime started on node nonode@nohost
2020-06-06 21:59:24.227 [info] <0.44.0> Application plug_crypto started on node nonode@nohost
2020-06-06 21:59:24.232 [info] <0.44.0> Application plug started on node nonode@nohost
2020-06-06 21:59:24.235 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-06-06 21:59:24.237 [info] <0.44.0> Application jsonapi started on node nonode@nohost
2020-06-06 21:59:24.237 [info] <0.44.0> Application prometheus_plugs started on node nonode@nohost
2020-06-06 21:59:24.280 [info] <0.44.0> Application conference started on node nonode@nohost
2020-06-06 22:00:01.505 [info] <0.44.0> Application lager started on node nonode@nohost
2020-06-06 22:00:01.520 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-06-06 22:00:01.548 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-06-06 22:00:01.560 [info] <0.44.0> Application tools started on node nonode@nohost
2020-06-06 22:00:01.566 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-06-06 22:00:01.572 [info] <0.44.0> Application recon started on node nonode@nohost
2020-06-06 22:00:01.716 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-06-06 22:00:01.716 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-06-06 22:00:01.725 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-06-06 22:00:01.729 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-06-06 22:00:01.740 [info] <0.44.0> Application connection started on node nonode@nohost
2020-06-06 22:00:01.747 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-06-06 22:00:01.756 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-06-06 22:00:01.762 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-06-06 22:00:01.775 [info] <0.44.0> Application eex started on node nonode@nohost
2020-06-06 22:00:01.787 [info] <0.44.0> Application telemetry started on node nonode@nohost
2020-06-06 22:00:01.797 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-06-06 22:00:01.798 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-06-06 22:00:01.866 [info] <0.44.0> Application jose started on node nonode@nohost
2020-06-06 22:00:01.866 [info] <0.44.0> Application joken started on node nonode@nohost
2020-06-06 22:00:01.880 [info] <0.44.0> Application mime started on node nonode@nohost
2020-06-06 22:00:01.883 [info] <0.44.0> Application plug_crypto started on node nonode@nohost
2020-06-06 22:00:01.886 [info] <0.44.0> Application plug started on node nonode@nohost
2020-06-06 22:00:01.888 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-06-06 22:00:01.890 [info] <0.44.0> Application jsonapi started on node nonode@nohost
2020-06-06 22:00:01.891 [info] <0.44.0> Application prometheus_plugs started on node nonode@nohost
2020-06-06 22:00:02.732 [info] <0.44.0> Application conference started on node nonode@nohost
2020-06-06 22:01:18.105 [info] <0.44.0> Application lager started on node nonode@nohost
2020-06-06 22:01:18.121 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-06-06 22:01:18.150 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-06-06 22:01:18.162 [info] <0.44.0> Application tools started on node nonode@nohost
2020-06-06 22:01:18.167 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-06-06 22:01:18.174 [info] <0.44.0> Application recon started on node nonode@nohost
2020-06-06 22:01:18.322 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-06-06 22:01:18.322 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-06-06 22:01:18.333 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-06-06 22:01:18.337 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-06-06 22:01:18.348 [info] <0.44.0> Application connection started on node nonode@nohost
2020-06-06 22:01:18.355 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-06-06 22:01:18.365 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-06-06 22:01:18.369 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-06-06 22:01:18.382 [info] <0.44.0> Application eex started on node nonode@nohost
2020-06-06 22:01:18.396 [info] <0.44.0> Application telemetry started on node nonode@nohost
2020-06-06 22:01:18.405 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-06-06 22:01:18.407 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-06-06 22:01:18.477 [info] <0.44.0> Application jose started on node nonode@nohost
2020-06-06 22:01:18.477 [info] <0.44.0> Application joken started on node nonode@nohost
2020-06-06 22:01:18.491 [info] <0.44.0> Application mime started on node nonode@nohost
2020-06-06 22:01:18.495 [info] <0.44.0> Application plug_crypto started on node nonode@nohost
2020-06-06 22:01:18.499 [info] <0.44.0> Application plug started on node nonode@nohost
2020-06-06 22:01:18.502 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-06-06 22:01:18.503 [info] <0.44.0> Application jsonapi started on node nonode@nohost
2020-06-06 22:01:18.504 [info] <0.44.0> Application prometheus_plugs started on node nonode@nohost
2020-06-06 22:01:18.548 [info] <0.44.0> Application conference started on node nonode@nohost
2020-06-06 22:02:27.590 [info] <0.44.0> Application lager started on node nonode@nohost
2020-06-06 22:02:27.604 [info] <0.44.0> Application xmerl started on node nonode@nohost
2020-06-06 22:02:27.631 [info] <0.44.0> Application sasl started on node nonode@nohost
2020-06-06 22:02:27.643 [info] <0.44.0> Application tools started on node nonode@nohost
2020-06-06 22:02:27.649 [info] <0.44.0> Application jsx started on node nonode@nohost
2020-06-06 22:02:27.660 [info] <0.44.0> Application ranch started on node nonode@nohost
2020-06-06 22:02:27.666 [info] <0.44.0> Application recon started on node nonode@nohost
2020-06-06 22:02:27.817 [info] <0.44.0> Application credentials_obfuscation started on node nonode@nohost
2020-06-06 22:02:27.817 [info] <0.44.0> Application rabbit_common started on node nonode@nohost
2020-06-06 22:02:27.824 [info] <0.44.0> Application amqp_client started on node nonode@nohost
2020-06-06 22:02:27.827 [info] <0.44.0> Application amqp started on node nonode@nohost
2020-06-06 22:02:27.834 [info] <0.44.0> Application poison started on node nonode@nohost
2020-06-06 22:02:29.463 [info] <0.44.0> Application prometheus started on node nonode@nohost
2020-06-06 22:02:29.463 [info] <0.44.0> Application prometheus_ex started on node nonode@nohost
2020-06-06 22:02:29.471 [info] <0.44.0> Application cowlib started on node nonode@nohost
2020-06-06 22:02:29.477 [info] <0.44.0> Application cowboy started on node nonode@nohost
2020-06-06 22:02:29.481 [info] <0.44.0> Application accept started on node nonode@nohost
2020-06-06 22:02:29.493 [info] <0.44.0> Application connection started on node nonode@nohost
2020-06-06 22:02:29.499 [info] <0.44.0> Application decimal started on node nonode@nohost
2020-06-06 22:02:29.509 [info] <0.44.0> Application db_connection started on node nonode@nohost
2020-06-06 22:02:29.515 [info] <0.44.0> Application postgrex started on node nonode@nohost
2020-06-06 22:02:29.528 [info] <0.44.0> Application eex started on node nonode@nohost
2020-06-06 22:02:29.541 [info] <0.44.0> Application telemetry started on node nonode@nohost
2020-06-06 22:02:29.552 [info] <0.44.0> Application ecto started on node nonode@nohost
2020-06-06 22:02:29.554 [info] <0.44.0> Application ecto_sql started on node nonode@nohost
2020-06-06 22:02:29.624 [info] <0.44.0> Application jose started on node nonode@nohost
2020-06-06 22:02:29.624 [info] <0.44.0> Application joken started on node nonode@nohost
2020-06-06 22:02:29.638 [info] <0.44.0> Application mime started on node nonode@nohost
2020-06-06 22:02:29.641 [info] <0.44.0> Application plug_crypto started on node nonode@nohost
2020-06-06 22:02:29.644 [info] <0.44.0> Application plug started on node nonode@nohost
2020-06-06 22:02:29.647 [info] <0.44.0> Application plug_cowboy started on node nonode@nohost
2020-06-06 22:02:29.648 [info] <0.44.0> Application jsonapi started on node nonode@nohost
2020-06-06 22:02:29.650 [info] <0.44.0> Application prometheus_plugs started on node nonode@nohost
2020-06-06 22:02:30.330 [info] <0.44.0> Application conference started on node nonode@nohost
2020-06-06 22:02:30.336 [error] <0.663.0> gen_server <0.663.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.336 [error] <0.664.0> gen_server <0.664.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.336 [error] <0.665.0> gen_server <0.665.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.336 [error] <0.666.0> gen_server <0.666.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.336 [error] <0.665.0> CRASH REPORT Process <0.665.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.336 [error] <0.667.0> gen_server <0.667.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.336 [error] <0.668.0> gen_server <0.668.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.337 [error] <0.637.0> gen_server <0.637.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.337 [error] <0.669.0> gen_server <0.669.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.337 [error] <0.670.0> gen_server <0.670.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.337 [error] <0.671.0> gen_server <0.671.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.337 [error] <0.638.0> gen_server <0.638.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.337 [error] <0.672.0> gen_server <0.672.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.337 [error] <0.662.0> Supervisor {<0.662.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.661.0>,3} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,3},{types,'Elixir.Postgrex.DefaultTypes'},{hostname,<<"localhost">>},{port,5432},{repo,...},...], <0.661.0>, #Ref<0.1871405841.752746497.194845>) at <0.665.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in context child_terminated
2020-06-06 22:02:30.338 [error] <0.667.0> CRASH REPORT Process <0.667.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.338 [error] <0.669.0> CRASH REPORT Process <0.669.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.338 [error] <0.670.0> CRASH REPORT Process <0.670.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.338 [error] <0.663.0> CRASH REPORT Process <0.663.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.338 [error] <0.639.0> gen_server <0.639.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.338 [error] <0.664.0> CRASH REPORT Process <0.664.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.338 [error] <0.673.0> gen_server <0.673.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.339 [error] <0.640.0> gen_server <0.640.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.339 [error] <0.641.0> gen_server <0.641.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.339 [error] <0.642.0> gen_server <0.642.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.339 [error] <0.643.0> gen_server <0.643.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.339 [error] <0.644.0> gen_server <0.644.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.340 [error] <0.662.0> Supervisor {<0.662.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.661.0>,5} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,5},{types,'Elixir.Postgrex.DefaultTypes'},{hostname,<<"localhost">>},{port,5432},{repo,...},...], <0.661.0>, #Ref<0.1871405841.752746497.194845>) at <0.667.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in context child_terminated
2020-06-06 22:02:30.340 [error] <0.673.0> CRASH REPORT Process <0.673.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.340 [error] <0.672.0> CRASH REPORT Process <0.672.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.340 [error] <0.674.0> gen_server <0.674.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.340 [error] <0.671.0> CRASH REPORT Process <0.671.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.340 [error] <0.641.0> CRASH REPORT Process <0.641.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.340 [error] <0.668.0> CRASH REPORT Process <0.668.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.341 [error] <0.666.0> CRASH REPORT Process <0.666.0> with 0 neighbours exited with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in 'Elixir.Connection':enter_stop/5 line 686
2020-06-06 22:02:30.341 [error] <0.645.0> gen_server <0.645.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.341 [error] <0.650.0> gen_server <0.650.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.341 [error] <0.646.0> gen_server <0.646.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.341 [error] <0.651.0> gen_server <0.651.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.341 [error] <0.652.0> gen_server <0.652.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.341 [error] <0.653.0> gen_server <0.653.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.342 [error] <0.662.0> Supervisor {<0.662.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.661.0>,7} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,7},{types,'Elixir.Postgrex.DefaultTypes'},{hostname,<<"localhost">>},{port,5432},{repo,...},...], <0.661.0>, #Ref<0.1871405841.752746497.194845>) at <0.669.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in context child_terminated
2020-06-06 22:02:30.342 [error] <0.675.0> gen_server <0.675.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.342 [error] <0.654.0> gen_server <0.654.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.342 [error] <0.655.0> gen_server <0.655.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.342 [error] <0.656.0> gen_server <0.656.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.342 [error] <0.657.0> gen_server <0.657.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.343 [error] <0.658.0> gen_server <0.658.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.343 [error] <0.659.0> gen_server <0.659.0> terminated with reason: #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399
2020-06-06 22:02:30.343 [error] <0.636.0> Supervisor {<0.636.0>,'Elixir.DBConnection.ConnectionPool.Pool'} had child {'Elixir.Postgrex.Protocol',<0.635.0>,5} started with 'Elixir.DBConnection.Connection':start_link('Elixir.Postgrex.Protocol', [{pool_index,5},{types,'Elixir.Postgrex.DefaultTypes'},{hostname,<<"localhost">>},{port,5432},{repo,...},...], <0.635.0>, #Ref<0.1871405841.752746498.188707>) at <0.641.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.RuntimeError',message => <<"connect raised KeyError exception: key :database not found. The exception details are hidden, as they may contain sensitive data such as database credentials. You may set :show_sensitive_data_on_connection_error to true when starting your connection if you wish to see all of the details">>} in 'Elixir.Keyword':'fetch!'/2 line 399 in context child_terminated
2020-06-06 22:02:31.051 [warning] <0.361.0> lager_error_logger_h dropped 147 messages in the last second that exceeded the limit of 50 messages/sec
